bayes_opt/bayesian_optimization.py

```diff
@@ -101,8 +110,23 @@ class BayesianOptimization(Observable):
     set_bounds()
         Allows changing the lower and upper searching bounds
     """
+
     def __init__(self, f, pbounds, random_state=None, verbose=2,
-                 bounds_transformer=None):
+                 bounds_transformer=None,
+                 dataset_path=None, output_path=None, target_column=None):
```

Comment:
`output_path` needs a different default value (I would suggest `'output'`), otherwise an error will be raised:
```
TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'
```
Moreover, if giving e.g. `output_path="foo"`, results are saved in a folder called `fooresults`. Besides the probably unintended behavior, this argument should be the actual folder in which files are stored. For instance, if giving `output_path='foo'`, files should be saved to `foo/exact_x.csv` etc, without any intermediate subfolder. Long story short, you can remove the `'results'` strings currently in the code and do just `os.path.join(self.output_path, 'exact_x.csv')` etc. (Note the use of `os.path.join()` instead of using `/` for generality with other Operative Systems.)

Reply:
fixed, now default output path is the working directory